GameData/RealismOverhaul/RO_SuggestedMods/Buran-Energia/ENERGIA.cfg

```diff
@@ -0,0 +1,1722 @@
+		}
+    }
+
+    @MODULE[ModuleGimbal]
+    {
+        %gimbalRange = 11.0
+        !gimbalRangeYP = NULL
+        !gimbalRangeYN = NULL
+        !gimbalRangeXP = NULL
+        !gimbalRangeXN = NULL
+        @gimbalResponseSpeed = 16
+    }
+
+    !MODULE[ModuleAlternator],*{}
+
+    !RESOURCE,*{}
+}
+
+
+
+
+
+
+
+@PART[Energia_RDTT1]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+
+    @MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+
+	%scale = 1.0
+	%rescaleFactor = 1.5625
+
+    @mass = 0.112
+    @crashTolerance = 10
+	@maxTemp = 873.15
+    %skinMaxTemp = 973.15
+
+    %engineType = RDTT
+    %engineTypeMult = 1
+    %massOffset = 0
+    %ignoreMass = False
+
+    @MODULE[ModuleEngines*]
+    {
+        @name = ModuleEnginesRF
+        @minThrust = 98.0
+        @maxThrust = 98.0
+        @heatProduction = 100
+        @useEngineResponseTime = False
+        @engineAccelerationSpeed = 0
+
+        @PROPELLANT[SolidFuel]
+        {
+            @name = HTPB
+        }
+
+        @atmosphereCurve
+        {
+            @key,0 = 0 205
+            @key,1 = 1 177
+        }
+    }
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 30.6
+		type = HTPB
+		basemass = -1
+	}
+
+    !RESOURCE,*{}
+}
+
+
+@PART[Energia_RDTT2]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+
+    @MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+
+	%scale = 1.0
+	%rescaleFactor = 1.5625
+
+    @mass = 0.112
+    @crashTolerance = 10
+	@maxTemp = 873.15
+    %skinMaxTemp = 973.15
+
+    %engineType = RDTT
+    %engineTypeMult = 1
+    %massOffset = 0
+    %ignoreMass = False
+
+    @MODULE[ModuleEngines*]
+    {
+        @name = ModuleEnginesRF
+        @minThrust = 49.0
+        @maxThrust = 49.0
+        @heatProduction = 100
+        @useEngineResponseTime = False
+        @engineAccelerationSpeed = 0
+
+        @PROPELLANT[SolidFuel]
+        {
+            @name = HTPB
+        }
+
+        @atmosphereCurve
+        {
+            @key,0 = 0 205
+            @key,1 = 1 177
+        }
+    }
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 15.6
+		type = HTPB
+		basemass = -1
+	}
+
+    !RESOURCE,*{}
+}
+
+
+@PART[Energia_RDTT3_left_Variant]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+
+    @MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+
+	%scale = 1.0
+	%rescaleFactor = 1.5625
+
+    @mass = 0.112
+    @crashTolerance = 10
+	@maxTemp = 873.15
+    %skinMaxTemp = 973.15
+
+    %engineType = RDTT
+    %engineTypeMult = 1
+    %massOffset = 0
+    %ignoreMass = False
+
+    @MODULE[ModuleEngines*]
+    {
+        @name = ModuleEnginesRF
+        @minThrust = 98.0
+        @maxThrust = 98.0
+        @heatProduction = 100
+        @useEngineResponseTime = False
+        @engineAccelerationSpeed = 0
+
+        @PROPELLANT[SolidFuel]
+        {
+            @name = HTPB
+        }
+
+        @atmosphereCurve
+        {
+            @key,0 = 0 205
+            @key,1 = 1 177
+        }
+    }
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 30.6
+		type = HTPB
+		basemass = -1
+	}

```

Comment:
Since you're defining multiple parts with the same sort of engine config... that's what the engine configs are meant to address, rather than cut-and-pasting blocks here.